crt_portal/cts_forms/views.py

```diff
@@ -11,16 +12,28 @@
 
 @login_required
 def IndexView(request):
-    latest_reports = Report.objects.order_by('-create_date')
+    # Sort data based on request from params, default to `created_date` of complaint
+    sort = request.GET.getlist('sort', ['create_date'])
     per_page = request.GET.get('per_page', 15)
-    paginator = Paginator(latest_reports, per_page)
     page = request.GET.get('page', 1)
 
-    latest_reports, page_format = pagination(paginator, page, per_page)
+    # Validate requested sort params
+    report_fields = [f.name for f in Report._meta.fields]
+    if all(elem.replace("-", '') in report_fields for elem in sort) is False:
+        raise Http404(f'Invalid sort request: {sort}')
```

Comment:
Is raising a 404 the intendend behavior we want, as opposed to gently dropping/ignoring the malformed request? This could be something the check in on quickly with product/design. (Not a blocker, just a question I'm curious about.)

Reply:
Users aren't going to be creating the URLs directly, so we will be masking problems as we develop pages with bad links if we are proactively masking problems. The default behavior is a 500, this will also give us an error message so we can fix the problem.